### PR TITLE
Move drop-refresh-spam to flag

### PR DESCRIPTION
**Pull request by dwwoelfel**
Moves [drop-refresh-spam](https://github.com/jsventures/instant/pull/1138) to a flag so we can easily turn it on and off.

### CODE REVIEW

File: server/flags-config/instant.perms.ts [hunk 1]
```diff
@@ -1,79 +1,91 @@
...
+    allow: {
+      view: "false",
+      create: "false",
+      delete: "false",
+      update: "false",
+    },
+  },
+};
+
+export default rules;
```
Comment (stopachka) [inline server/flags-config/instant.perms.ts:91]: awwh yeahh : )

File: server/flags-config/instant.schema.ts [hunk 2]
```diff
@@ -1,50 +1,50 @@
-// instant-config
-// https://instantdb.com/dash?s=main&t=home&app=24a4d71b-7bb2-4630-9aee-01146af26239
+// http://localhost:3000/dash?s=main&t=home&app=24a4d71b-7bb2-4630-9aee-01146af26239
```
Comment (stopachka) [inline server/flags-config/instant.schema.ts:1]: (Looking here, I am going to just nix this url in the generation; I don't think it's too helpful)

**Review by stopachka — APPROVED** [review]
niice!

File: server/flags-config/instant.schema.ts (hunk 2 above)
Comment (dwwoelfel) [reply server/flags-config/instant.schema.ts:1]: It seems kind of useful to me! Just a little annoying for us since we use dev and prod.

**Pull request merged**